configure.ac

```diff
@@ -1859,6 +1871,8 @@ AC_DEFINE_UNQUOTED(SYSTEMD_JOURNAL_MODE, `journald_mode`, [Systemd-journal suppo
 AC_DEFINE_UNQUOTED(HAVE_INOTIFY, `enable_value $ac_cv_func_inotify_init`, [Have inotify])
 AC_DEFINE_UNQUOTED(ENABLE_PYTHONv2, `enable_value $enable_python2`, [Enable Python 2 support])
 AC_DEFINE_UNQUOTED(ENABLE_PYTHONv3, `enable_value $enable_python3`, [Enable Python 3 support])
+AC_DEFINE_UNQUOTED(ENABLE_PYTHONv2_AND_v3, `enable_value $enable_python2_and_3`, [Enable both Python 2 and Python 3 support])
```

Comment:
I chose this method to avoid additional ifdefing in our code base (it's already complex enough there), so I can compile the same library twice setting the appropriate macro in Makefile (`SYSLOG_NG_ENABLE_PYTHONv{2,3}`).